django/thunderstore/core/mixins.py

```diff
@@ -106,3 +107,14 @@ class Meta:
         indexes = [
             models.Index(fields=["last_modified"]),
         ]
+
+
+class AdminMixin(models.Model):
```

Comment:
This should probably be called `AdminLinkMixin` as just `AdminMixin` is super ambiguous in the Django world